### ISSUE TITLE
Guangdong - West African peace conference seems to cause crash

### ISSUE THREAD

**Issue by Estonland** [TNODevs/TNO-Bug-Tracker#1538]
### Operating System

Windows 10

### HOI4 version

1.12.8

### TNO version

1.4.0

### Do you have other mods in use?

Toolpakt, Radio TNO, The Last Waltz Extended

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[game.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10609044/game.log)
[error.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10609046/error.log)

### Describe the crash

I was playing a game as Guangdong with no prior issues, however my game crashes just as the peace deal for Free France winning the West African War fires. I have loaded three times, and in each attempt the game crashed while trying to implement the peace deal.

### Steps to reproduce / Cause of the crash

1. Start as Guangdong
2. Play until Free France wins the West African War

**Comment by Elucidate1000** [comment]
Seconded. My LBJ run crashed as soon as I won the WAW.
[error.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10609686/error.log)
[game.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10609687/game.log)

**Comment by Randarkman** [comment]
Same with me, playing Guangdong (I assume it doesn't matter who), Free France wins, peace conference loads, crash. Tried 4 times now.
[error.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10610759/error.log)
[game.log](https://github.com/TNODevs/TNO-Bug-Tracker/files/10610760/game.log)

**Comment by Randarkman** [comment]
If I were to guess this is the relevant line from the error log 
[11:54:50][effectimplementation.cpp:677]: common/scripted_effects/TNO_WestAfrica_Crisis_scripted_effects.txt:946: promote_character: Cannot promote a charater with several country leader roles without without an ideology in input

**Comment by Randarkman** [comment]
So yeah, checked above and it's something in the entry for the effects relating to Susu in a Free French vicotry, and I think specifically for Louis Lansana Beavogui.

**Comment by runelead00** [comment]
#1509 Please upload save files for peace deal crashes for all the people above

**Comment by runelead00** [comment]
also which scenario was this crash on and what did borders look like immediately before the crash

**Comment by bigbadboy78** [comment]
Can confirm. Playing Guang and game crashed

**Comment by MeridianPL** [comment]
![Zrzut ekranu (287)](https://user-images.githubusercontent.com/118394495/217070360-b1e590bc-429e-4df8-be6e-2f4c07244566.png)
![Zrzut ekranu (288)](https://user-images.githubusercontent.com/118394495/217070365-8565b1a9-1fc5-4ab7-9868-88b28f5e3324.png)
The same thing happened to me, and I managed to get a screenshot of the borders and general state of the war right as the crash was happening, hoping this helps.

**Comment by bigbadboy78** [comment]
Yeah, I tag switched and the victory goes to peace conference. Also need to beat all of the factions even if you beat faction leader to capitulate them.

**Comment by TheUndeadCommissar** [comment]
The same thing happened to my right-wing Komi game.
I was in regional around June, maybe July 1968 and my game was finishing a peace conference which I assumed was the west African war. (Free France lost btw, I don't know if that affected it) My game crashed, and I reloaded and it crashed again and I gave up trying to play. I just thought that it was my computer but I have a pretty decent computer.
